dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -162,48 +185,35 @@ private Response postWithEndpointState(String endpointId, String url, MediaType
      */
 
     @POST
-    @Path("/generateintegritydata/{params:.*}")
-    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("/_generate")
     @Produces("text/plain")
-    public Response generateIntegrityData(@Context HttpServletRequest request, @FormDataParam("AUTH_TOKEN") String auth_token_digest)  {
-
-        String remoteIP = null;
-        try {
-
-            if ( !UtilMethods.isSet( auth_token_digest ) ) {
-                return Response.status( HttpStatus.SC_BAD_REQUEST ).entity( "Error: 'endpoint' is a required param." ).build();
-            }
+    public Response generateIntegrityData(@Context HttpServletRequest request)  {
 
+        String remoteIP = getRemoteIP(request);
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)